starter-repos/app-template/validation/functions/example-validator-kubeval.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
```

Comment:
Can we rename this file to `kubeval.yaml`?

Reply:
DONE. Renamed to `kubeval-validator.yaml`.